<ui version="4.0" >
 <class>IntervalView</class>
 <widget class="QWidget" name="IntervalView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QPushButton" name="addPushButton" >
          <property name="icon" >
           <iconset>
            <normaloff>:table_add</normaloff>:table_add</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePushButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff>:table_delete</normaloff>:table_delete</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rowUpPushButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../resources/application.qrc" >
            <normaloff>:/table_row_up</normaloff>:/table_row_up</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rowDownPushButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../resources/application.qrc" >
            <normaloff>:/table_row_down</normaloff>:/table_row_down</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>208</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="cornerButtonEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Interval type</string>
           </property>
           <property name="buddy" >
            <cstring>intervalTypeComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3" >
          <layout class="QHBoxLayout" name="horizontalLayout_3" >
           <item>
            <widget class="QComboBox" name="intervalTypeComboBox" />
           </item>
           <item>
            <widget class="QToolButton" name="intervalTypeToolButton" >
             <property name="text" >
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Distance</string>
           </property>
           <property name="buddy" >
            <cstring>distanceDoubleSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="distanceDoubleSpinBox" >
           <property name="suffix" >
            <string> km</string>
           </property>
           <property name="decimals" >
            <number>3</number>
           </property>
           <property name="maximum" >
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Duration</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <widget class="QTimeEdit" name="durationTimeEdit" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Computed pace</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3" >
          <widget class="QLineEdit" name="paceLineEdit" >
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>232</red>
                 <green>232</green>
                 <blue>232</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>232</red>
                 <green>232</green>
                 <blue>232</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>232</red>
                 <green>232</green>
                 <blue>232</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>232</red>
                 <green>232</green>
                 <blue>232</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>232</red>
                 <green>232</green>
                 <blue>232</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>232</red>
                 <green>232</green>
                 <blue>232</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font" >
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Notes</string>
           </property>
           <property name="buddy" >
            <cstring>notesPlainTextEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3" >
          <widget class="QPlainTextEdit" name="notesPlainTextEdit" />
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/application.qrc" />
 </resources>
 <connections/>
</ui>
